src/libimg/Image.cpp

```diff
@@ -275,6 +277,227 @@ void Image::drawImage(const Image& src, int dstX, int dstY) {
     }
 }
 
+void Image::plot(int x, int y, float brightness) {
+    auto alpha = (int)(brightness * 255);

```

Comment:
I personally prefer to use `auto` only if it makes the code more readable. In this situation, I think a simple `int alpha = brightness * 255` would be more readable, but that might be a personal preference.